clients/extension/styles/lighthouse.css

```diff
@@ -46,96 +46,65 @@ html, body {
   margin-right: 16px;
 }
 
-.header-titles {
-  display: flex;
-  flex-direction: column;
-  align-self: center;
-  flex: 1;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.button {
+  cursor: pointer;
+  font-family: 'Roboto', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  font-weight: 500;
+  font-size: 12px;
+  border: none;
+  color: #fff;
+  background-color: #2979ff;
+  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
+  border-radius: 2px;
+  padding: 7px 12px 5px 12px
 }
 
-.header-titles__main {
-  margin: 0;
-  font-family: 'Roboto', Arial, sans-serif;
-  color: currentColor;
-  font-weight: 300;
-  margin-bottom: 4px;
+.button:disabled {
+  background-color: #cccccc;
+  color: #666666;
+  cursor: not-allowed;
 }
 
-.header-titles__url {
-  margin: 0;
-  font-size: 14px;
-  opacity: 0.74;
-  color: currentColor;
-  font-family: 'Roboto', Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  font-weight: 400;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+.main {
+  padding: 16px;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
 }
 
-.header-titles__throttling {
-  position: relative;
-  margin-top: 5px;
-  padding-left: 20px;
+.section {
+  place-self: center;
 }
 
-.header-titles__throttling input {
-  position: absolute;
-  left: 0;
+.section:first-child {
+  border-bottom: solid 1px lightgray;
 }
 
-.main {
-  height: 70px;
+.section--run {
   display: flex;
-  padding: 16px;
-  background-color: var(--lh-blue-color);
-  justify-content: space-around;
-  align-items: flex-end;
+  position: relative;
 }
 
-.button {
-  cursor: pointer;
-  font-family: 'Roboto', Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  font-weight: 500;
-  font-size: 12px;
-  border: none;
-  color: #fff;
-  background-color: #2979ff;
-  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
-  border-radius: 2px;
-  padding: 7px 12px 5px 12px
+.section--devtools-info {
+  font-size: 16px;
+}
+
+.button--generate__wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
 }
 
 .button--generate {
+  height: 40px;
```

Comment:
lets use hwi's mock

![image](https://user-images.githubusercontent.com/39191/62739300-553b9f00-b9e9-11e9-8386-4c67e4569a12.png)

i really like the logo layout etc.


we can add a settings cog next to 'run lighthouse'. and you can put the 'Uses the PageSpeed Insights API` text within options.